src/main/java/com/twilio/oai/resolver/csharp/CsharpCodegenModelDataTypeResolver.java

```diff
@@ -0,0 +1,61 @@
+package com.twilio.oai.resolver.csharp;
+
+import com.twilio.oai.StringHelper;
+import com.twilio.oai.common.ApplicationConstants;
+import com.twilio.oai.resolver.IConventionMapper;
+import com.twilio.oai.resolver.common.CodegenModelDataTypeResolver;
+import org.openapitools.codegen.CodegenProperty;
+
+import java.util.Map;
+import java.util.Optional;
+
+// Overriding default behavior and adding "x-jsonConverter" to enum
+public class CsharpCodegenModelDataTypeResolver extends CodegenModelDataTypeResolver {
+    private final IConventionMapper mapper;
+
+    public CsharpCodegenModelDataTypeResolver(IConventionMapper mapper, Map<String, String> modelFormatMap) {
+        super(mapper, modelFormatMap);
+        this.mapper = mapper;
```

Comment:
The parent class already stores the mapper. Should make it `protected` if it's needed in the child class. Then this constructor can be deleted.

Reply:
removed mapper, but constructor is needed to call parent constructor.